atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,45 @@
+package br.edu.uepb.eda.atividade03;
+
+public class PilhaComLista implements Pilha_IF {
+
+	protected ListaEncadeada pilha = new ListaEncadeada();
+
+	public void push(Integer element) throws Exception {
+		if(this.isFull()) {
+			throw new Exception("LISTA CHEIA");
+		}else {
+			pilha.insert(element);
+		}
+	}
+
+	public Integer pop() throws Exception {
+		if(this.isEmpty()) {
+			throw new Exception("LISTA VAZIA");
+		}else {
+			int aux = pilha.valor;
+			pilha.remove(pilha.valor);
+			return aux;
+		}
+	}
+
+	public Integer top() throws Exception {
+		if(this.isEmpty()) {
+			throw new Exception("LISTA VAZIA");
+		}else {
+			return pilha.valor;
+		}
+	}
+
+	public boolean isEmpty() {
+		return pilha.isEmpty();
+	}
+
+	public boolean isFull() {
+		if(pilha.size() == 10) {
```

Comment:
Apenas um ponto de atenção, cuidado com magic number ou valores constantes soltos pelo código, o ideal é levar esse valor para uma variável que represente o contexto daquela informação, por exemplo:

primeiro declara a variável constante com um nome que descreve o valor que ela receber
```
private final int MAX_ELEMENT = 10;
```

```suggestion
		if(pilha.size() == MAX_ELEMENT) {
```